lib/compat/wordpress-6.3/script-loader.php

```diff
@@ -0,0 +1,15 @@
+<?php
+/**
+ * Updates the script-loader.php file
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Removes duotone SVG filter presets from all being output.
```

Comment:
```suggestion
 * Prevent unused duotone SVG filter presets from being output.
```

It could be the hour of the day but I had to read this sentence twice before understanding its intended implication. Maybe we could reword it?

Reply:
I can see how that line is confusing. These remove the action that outputs all of the SVGs, so that the WP_Duotone_Gutenberg class can handle outputting only the used SVGs. I'll update the comment to explain this more clearly and maybe that'll help give more context to make it clearer. Happy to keep tweaking this though!